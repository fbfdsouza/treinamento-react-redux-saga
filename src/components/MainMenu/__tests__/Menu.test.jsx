import React from 'react';
import { shallow } from 'enzyme';
import Menu from '../Menu';

const minimalProps = {
  children: <div />
};

describe('<Menu /> Tests', () => {
  it('should render without exploding', done => {
    const wrapper = shallow(<Menu {...minimalProps} />);
    expect(wrapper).toHaveLength(1);
    done();
  });

  it('should render correctly', done => {
    const wrapper = shallow(<Menu {...minimalProps} />);
    expect(wrapper).toMatchSnapshot();
    done();
  });
});
