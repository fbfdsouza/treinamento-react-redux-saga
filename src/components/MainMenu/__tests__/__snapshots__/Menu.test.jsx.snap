// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> Tests should render correctly 1`] = `
<ul
  className="root"
>
  <div />
</ul>
`;
